atomic_reactor/plugin.py

```diff
@@ -192,19 +192,18 @@ def run(self, keep_going=False):
                 raise PluginFailedException(msg)
 
             plugin_conf = plugin_request.get("args", {})
+            run_fail = False
             try:
                 plugin_class = self.plugin_classes[plugin_name]
             except KeyError:
-                self.on_plugin_failed()
-
                 if plugin_request.get('required', True):
```

Comment:
yeah I just realized that is_allowed_to_fail is used when plugin is already running